src/learning/vector-utils.ts

```diff
@@ -0,0 +1,123 @@
+
+import { FastPotential } from '..'
```

Comment:
nit: remove line at top
```suggestion
import { FastPotential } from '..'
```